### PR TITLE
Modified openshift/README.md with more FAQs with issues faced during on boarding

### PR DESCRIPTION
**Pull request by dgpatelgit**
Modified openshift/README.md's FAQ section with some issue faced during on boarding.

### CODE REVIEW

File: openshift/README.md [hunk 1]
```diff
@@ -92,6 +92,42 @@ to remove the OpenShift project and all allocated AWS resources.
...
+**resolution**: Always ensure that your OC_TOKEN value in env.sh is latest as per dev-cluster console value. Get the latest value from dev-cluster console.
+
+4. AWS console does not show my RDS / DynamoDB / DataPipe lines.
+
+**cause**: AWS region is wrong in AWS console.
+**resolution**: Ensure that you have choosen right region on top-right corner of AWS console, it should be 'N. Virginia (us-east-1)' for all our development resources.
+
+5. Enable to create dev cluster project / RDS database with correct prefixes.
+
+**cause**: When Kurboros Id and Github ids are not same, we might face this issue. In env.sh we set USER_ID to kurboros id, but this value gets overwritten by git user id during delpoy.sh execution. THIS IS REQUIRED ONLY WHEN GITHUB AND KURBOROS IDS ARE DIFFERENT.
```
Comment (deepak1725) [inline openshift/README.md]: ~~KURBOROS~~ KERBEROS

File: openshift/README.md [hunk 2]
```diff
@@ -92,6 +92,42 @@ to remove the OpenShift project and all allocated AWS resources.
...
+**resolution**: Hardcode the OC_PROJECT and RDS_INSTANCE_NAME with fixed value for ${USER_ID} fields.
+Example: In env.sh, set
+    OC_PROJECT=dhpatel-fabric8-analytics
+    RDS_INSTANCE_NAME=dhpatel-bayesiandb
```
Comment (deepak1725) [inline openshift/README.md]: don't put your KERBEROS anywhere.. We can use {KERBEROS} as a placeholder

**Review by deepak1725 — CHANGES_REQUESTED** [review]

File: openshift/README.md (hunk 1 above)
Comment (dgpatelgit) [reply openshift/README.md]: Comment accepted and fixed.

File: openshift/README.md (hunk 2 above)
Comment (dgpatelgit) [reply openshift/README.md]: Comment accepted and fixed.

File: openshift/README.md [hunk 3]
```diff
@@ -89,9 +89,43 @@ to remove the OpenShift project and all allocated AWS resources.
 1. In dev_console, `bayesian-data-importer` service is down.
 
 **cause**: In this case you have some data messed up in your Dynamo DB Tables.
-**resolution**: Completely remove **your tables only** from AWS Dynamo DB, i.e  tables prefixed with name `your_kerboros_*`. 
+**resolution**: Completely remove **your tables only** from AWS Dynamo DB, i.e  tables prefixed with name `your_kerberos_*`.
```
Comment (deepak1725) [inline openshift/README.md]: Use Same naming convention everywhere. 
If you are using {`YOUR_KERBEROS_ID`}, lets stick with it.

File: openshift/README.md [hunk 4]
```diff
@@ -89,9 +89,43 @@ to remove the OpenShift project and all allocated AWS resources.
...
+    dynamodb-secret-access-key: <KEY>
+    sqs-access-key-id: <ID>
+    sqs-secret-access-key: <KEY>
+
+Note: Replace <ID> and <KEY> with the id and key values present in other keys of same secret.
+
+Manually add below key in 'aws-dynamodb' secrets:
+    aws_region: dXMtZWFzdC0x
+
+Note: Value for this key is BASE64 encoded string of AWS region, in above case its 'us-east-1'
```
Comment (deepak1725) [inline openshift/README.md]: Add cli option to encode values..
```
$ base64 <<< 'us-east-1'
```

File: openshift/README.md [hunk 5]
```diff
@@ -89,9 +89,43 @@ to remove the OpenShift project and all allocated AWS resources.
...
+
+3. Upon running ./deploy.sh or any script from terminal throws INVALID TOKEN error.
```
Comment (deepak1725) [inline openshift/README.md]: `./deploy.sh` : Wrap in Code Strings to highlight it

File: openshift/README.md [hunk 6]
```diff
@@ -89,9 +89,43 @@ to remove the OpenShift project and all allocated AWS resources.
...
+
+**cause**: If dev-cluter session is timeout / user logs into again, then it generated a new login token for CLI.
+**resolution**: Always ensure that your OC_TOKEN value in env.sh is latest as per dev-cluster console value. Get the latest value from dev-cluster console.
+
+4. AWS console does not show my RDS / DynamoDB / DataPipe lines.
```
Comment (deepak1725) [inline openshift/README.md]: `DataPipe lines.` has nothing to do with dev_cluster deployments. You may remove it

File: openshift/README.md [hunk 7]
```diff
@@ -89,9 +89,43 @@ to remove the OpenShift project and all allocated AWS resources.
...
+
+Manually add below key in 'aws-dynamodb' secrets:
+    aws_region: dXMtZWFzdC0x
+
+Note: Value for this key is BASE64 encoded string of AWS region, in above case its 'us-east-1'
+
+3. Upon running ./deploy.sh or any script from terminal throws INVALID TOKEN error.
+
+**cause**: If dev-cluter session is timeout / user logs into again, then it generated a new login token for CLI.
+**resolution**: Always ensure that your OC_TOKEN value in env.sh is latest as per dev-cluster console value. Get the latest value from dev-cluster console.
```
Comment (deepak1725) [inline openshift/README.md]: OC_TOKEN -> `OC_TOKEN`

File: openshift/README.md [hunk 8]
```diff
@@ -89,9 +89,43 @@ to remove the OpenShift project and all allocated AWS resources.
...
+**cause**: AWS region is wrong in AWS console.
+**resolution**: Ensure that you have choosen right region on top-right corner of AWS console, it should be 'N. Virginia (us-east-1)' for all our development resources.
+
+5. Enable to create dev cluster project / RDS database with correct prefixes.
+
+**cause**: When kerberos Id and Github ids are not same, we might face this issue. In env.sh we set USER_ID to kerberos id, but this value gets overwritten by git user id during delpoy.sh execution. THIS IS REQUIRED ONLY WHEN GITHUB AND KERBEROS IDS ARE DIFFERENT.
+**resolution**: Hardcode the OC_PROJECT and RDS_INSTANCE_NAME with fixed value for ${USER_ID} fields.
+Example: In env.sh, set
+    OC_PROJECT={YOUR_KERBEROS_ID}-fabric8-analytics
+    RDS_INSTANCE_NAME={YOUR_KERBEROS_ID}-bayesiandb
```
Comment (deepak1725) [inline openshift/README.md:127]: Add a PostScript message: Jump to `Monitoring` tab to see failure messages.

**Review by deepak1725 — CHANGES_REQUESTED** [review]

File: openshift/README.md (hunk 3 above)
Comment (dgpatelgit) [reply openshift/README.md]: This was existing documentation, I just corrected spelling here. However, now modified to have uniform name.

File: openshift/README.md (hunk 4 above)
Comment (dgpatelgit) [reply openshift/README.md]: Added shell script for this.

File: openshift/README.md (hunk 5 above)
Comment (dgpatelgit) [reply openshift/README.md]: Code Modified.

File: openshift/README.md (hunk 6 above)
Comment (dgpatelgit) [reply openshift/README.md]: Section #4, deleted from this readme

File: openshift/README.md (hunk 7 above)
Comment (dgpatelgit) [reply openshift/README.md]: Highlighting done.

File: openshift/README.md (hunk 8 above)
Comment (dgpatelgit) [reply openshift/README.md:127]: Added a note at the bottom as discussed on Slack.

**Review by deepak1725 — APPROVED** [review]
LGTM.

**Pull request merged**